scim2-sdk-common/src/main/java/com/unboundid/scim2/common/utils/ScimJsonFactory.java

```diff
@@ -68,7 +69,8 @@ protected ScimJsonFactory(@NotNull final ScimJsonFactory sourceFactory,
   JsonParser createScimFilterParser(@NotNull final Reader r)
       throws IOException
   {
-    IOContext ctxt = _createContext(r, false);
+    ContentReference reference = ContentReference.construct(true, r);
+    IOContext ctxt = _createContext(reference, false);
```

Comment:
This is an unrelated change that resolves another deprecation warning. I've kept this in a separate commit.